### PR TITLE
Add Kochi Wordcamp and ReactFoo Delhi

### PR DESCRIPTION
**Pull request by sivaram636**
**Details of the conference**:

- Website:https://2019.kochi.wordcamp.org
- Date: 1 DEC
- Venue: Seminar complex, CUSAT, Kochi
- Description: Primarily a Wordpress Conference, covering everything related to WordPress and include things such as blogging, coding, branding, and security.

### CODE REVIEW

**Comment by sivaram636** [comment]
https://2019.kochi.wordcamp.org/call-for-speakers/ @nikhita CFP

File: README.md [hunk 1]
```diff
@@ -8,8 +8,10 @@ To add a conference to the list, please [contribute](#contributing)!
 
 | Conference | Date | Venue | Description | Scholarship |
 |------------|------|-------|-------------|-------------|
+| [WordCamp Kochi](https://2019.kochi.wordcamp.org/) | 1 DEC | SEMINAR COMPLEX CUSAT, KOCHI | Primarily a Wordpress Conference, covering everything related to WordPress and include things such as blogging, coding, branding, and security. | No |
```
Comment (nikhita) [inline README.md]:
```suggestion
| [WordCamp Kochi](https://2019.kochi.wordcamp.org/) | 1 Dec | Seminar Complex, Cusat, Kochi | Primarily a Wordpress Conference, covering everything related to WordPress and include things such as blogging, coding, branding, and security. | No |
```

Comment (sivaram636) [reply README.md]: @nikhita updated :)

File: README.md [hunk 2]
```diff
@@ -8,8 +8,10 @@ To add a conference to the list, please [contribute](#contributing)!
 
 | Conference | Date | Venue | Description | Scholarship |
 |------------|------|-------|-------------|-------------|
+| [WordCamp Kochi](https://2019.kochi.wordcamp.org/) | 1 DEC | Seminar Complex Cusat, Kochi | Primarily a Wordpress Conference, covering everything related to WordPress and include things such as blogging, coding, branding, and security. | No |
```
Comment (nikhita) [inline README.md]: Please also change `DEC` to `Dec` :)

Comment (sivaram636) [reply README.md]: Done

**Pull request merged**